Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as

```diff
@@ -564,13 +562,25 @@ package com.flashlight.vnc
 				logger.info("event.ctrlKey "+event.ctrlKey);
 				logger.info("event.keyLocation "+event.keyLocation);
 				logger.info("event.shiftKey "+event.shiftKey);
-				logger.info("expectKeyInput "+expectKeyInput);
 
 				// Ignore Ctrl-v, onTextInput() handles this
 				if (event.ctrlKey && event.keyCode == Keyboard.V) {
 					return;
 				}
 
+				// Send other ctrl commands
+				if(event.ctrlKey && event.type == flash.events.KeyboardEvent.KEY_DOWN && event.keyCode != 17 && event.keyCode != 16) {
```

Comment:
It's shift and ctrl, I'll change to constants.
It's so we send characters if there's really a character there, instead of invoking the function with unexpected arguments, also may use range here so it's more clear what's happening.